textproto/test.ts

```diff
@@ -189,3 +189,66 @@ Deno.test({
     assertEquals(line, input);
   },
 });
+
+/*
+Test: 1 null
```

Comment:
How do these work? What are the expected/actual values? I'm assuming these are valuable tests, so I'm interested in having them run on `Deno.test` like the rest of the test suite. As per your other comment, I understand you're unfamiliar with Deno's test runner; Feel free to contact me directly by email or on Discord, or ask around on our community server at https://discord.gg/deno

Reply:
This is a comment that I added accidentally, my bad. Basically @Zamralik designed a test to compare the output of the old code to the output of the new code to ensure behavior was the same. I am adapting those test to verify that the output stays the same. This means I will input some determined values and compare the output to a list of expected determined values.
Those take time to write, if it should be part of this PR, then I will need some time to adapt them.

Reply:
You can remove these comments now the tests have been added.

Reply:
Good catch, just did it.